### ISSUE TITLE
Silver Arrows

### ISSUE THREAD

**Issue by Stargentian** [AlexModGuy/Ice-and-Fire-Suggestions#118]
Posting on behalf of Corvus because Github isn't working for them right now.
we need SILVER ARROWS. extra damage towards undead. please and thank you.

**Comment by Auphe** [comment]
Yes please. More use for silver

**Comment by LordDecay26** [comment]
yes, i believe every weapon we have made out of normal materials such as iron and diamond should also have a silver variant, we need silver ranged weapons, so, silver arrows is the answer. but also maybe crossbows? naaahhh... but maybe? :D
